aikau/src/main/resources/webscript-libs/doclib/doclib.lib.js

```diff
@@ -577,7 +577,8 @@ function getDocLibTree(options) {
                   siteId: options.siteId,
                   containerId: options.containerId,
                   rootNode: options.rootNode,
-                  rootLabel: options.rootLabel || "documentlibrary.root.label"
+                  rootLabel: options.rootLabel || "documentlibrary.root.label",
+                  useHash: (options.useHash !== false)
```

Comment:
This is the same as all the other places that options.useHash is used (e.g in the breadcrumb trail and the document list)... we want the default to be true, so yes... useHash could be undefined in which case we are happy that useHash is set to true.